alws/utils/pulp_utils.py

```diff
@@ -9,13 +9,83 @@
     CoreContent,
     CoreRepository,
     CoreRepositoryContent,
+    RpmModulemd,
+    RpmModulemdPackages,
     RpmPackage,
 )
 
 
 def get_uuid_from_pulp_href(pulp_href: str) -> uuid.UUID:
     return uuid.UUID(pulp_href.split("/")[-2])
 
+# TODO: Merge with get_rpm_packages_from_repository
+# or keep separate functions but make them share code
```

Comment:
I suppose we can merge them

Reply:
Not merging them as we will revisit/refactor this in the near future